x/emissions/keeper/inference_synthesis/synth_palette_weight.go

```diff
@@ -99,31 +111,48 @@ func (p *SynthPalette) CalcWeightedInference(weights RegretInformedWeights) (Inf
 	sumWeights := alloraMath.ZeroDec()
 	err := error(nil)
 
-	for _, inferer := range p.Inferers {
-		runningUnnormalizedI_i, sumWeights, err = AccumulateWeights(
-			p.InferenceByWorker[inferer],
-			weights.inferers[inferer],
-			p.InfererRegrets[inferer].noPriorRegret,
-			p.AllInferersAreNew,
-			runningUnnormalizedI_i,
-			sumWeights,
-		)
+	// If there is only one not-new inferer, then just consider that inferer
+	if p.SingleInfererNotNew != "" {
```

Comment:
it's not wrong this way but I would prefer to use the booleans as booleans as opposed to derivative properties such as this (this is only populated if those booleans are of a certain value)

Reply:
Yeah, also thought about this but was opposed to adding a new property for the palette just for that... maybe it's a better idea to add...